doc/StartScriptFormat.txt

```diff
@@ -50,6 +50,8 @@
 	NumTeams=y;         // same here, set this to check if the script is right
 	NumAllyTeams=z;     // see above
 
+	FixedSeed = 1; //Initialize the synced RNG with a specific seed for reproducible benchmarking runs only
```

Comment:
Why 'benchmarking runs only'? Surely it's useful for things like say reproducible cutscenes in missions.

Reply:
fixed - check out the new description